content/blog/2024/01/2024-01-17-jenkins-contributor-summit-update.adoc

```diff
@@ -0,0 +1,41 @@
+---
+layout: post
+title: "2024 Jenkins Contributor Summit - Update"
+tags:
+- events
+- community
+- contribute
+author: jmMeessen
+discourse: true
+opengraph:
+  image: /images/post-images/2023/11/ContribSummit/Contributor_summit_2024.png
+description: >
+  We are looking forward to meet you during the  Jenkins Contributor Summit in Brussels prior to FOSDEM.
+---
+
+The Jenkins Contributor Summit in Brussels (February 2nd) is now quickly approaching. We are looking forward to meeting you.
+
+If you plan to join us and didn’t let us know yet, don’t forget to register by dropping us a note. The list of registered attendees can be found in this document: link:https://docs.google.com/spreadsheets/d/1fatDxa39U-yHW6iTpMQZuW_pMf8G9jhO6yczfoqcY08/edit?usp=sharing[Contributor Summit 24 - Attendees].
+
+The Summit will take place at link:https://www.betacowork.com/[**Betacowork**], located at *4 Rue des Pères Blancs, 1040 Brussels, Belgium* (link:https://maps.app.goo.gl/S8VrWsrErmLMXqza7[Betacowork] on Google Maps) in room "AZZAR"
+
+Find hereafter a quick overview of the day’s planning:
+
+* We will welcome attendees from 9h30 (9:30 AM) on
+* The Summit will start at 10h00 (10:00 AM)
+* During the morning we will listen to the introduction by the various Jenkins Officers. 
+  Each one will present, for his domain of responsibility, the year’s projects and goals, the challenges, as well as the topics that could/should be discussed during the Summit.
+* At 12h00 (12:00 AM) we will have a one hour lunch break where people can go grab some food (sandwiches) at a nearby shop.
+* At 13h00 (1:00 PM), we start the afternoon sessions and discussions:
+** Changes made to Jenkins to support the new CB HA solution (Vincent L.)
```

Comment:
I think that many readers may not recognize "CB" as CloudBees and some readers may not recognize "HA" and "high  availability".  I think it is better to expand the acronyms.
```suggestion
** Changes made to Jenkins to support the new CloudBees high availability solution (Vincent L.)
```